stores-api.yaml

```diff
@@ -605,21 +610,11 @@ components:
           postalCode: "10967"
           city: Berlin
           state: ""
-        neighborhood:
-        - Kreuzberg
-        - Neukölln
-        - Körtekiez
-        geoLocation:
-          type: Point
-          coordinates:
-          - 13.408134
-          - 52.489908
-        profileImageURL: https://7cbf5172f31e91cb7a5d-d8e96be60ea1fd7ad98fb07a482142ee.ssl.cf3.rackcdn.com/store-front-X8l4AvUY.jpg
         profileVideoURL: https://youtu.be/PBkb2mzbqKE
-        shopOwnerImage: https://drive.google.com/file/d/1Ls5Yjc760oEfnibC9Xv29OtALtmV_Ss4/view?usp=sharing
```

Comment:
What is the new field for uploading the image?

Reply:
You upload via POST stores/{id}/image